src/preset.ts

```diff
@@ -868,7 +869,7 @@ export async function expandConfigurePreset(folder: string, name: string, worksp
                     shell: true
                 }).result;
 
-                if (!compilerLocation.stdout) {
+                if (vscode.workspace.getConfiguration('cmake').get('enableVSDevEnv')) {
```

Comment:
@gcampbell-msft is it okay to call getconfig inline like this?

Reply:
Yes, this is the way to do this. 👍